cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/tests/tests.rs

```diff
@@ -182,6 +183,21 @@ mod bridge_hub_westend_tests {
 		>(collator_session_keys(), bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID)
 	}
 
+	#[test]
```

Comment:
@claravanstaden @alistair-singh  could you, please, add here some complex test covering all extrinsics needed for the relayer, that are ok and working as expected? we have here similar `complex_relay_extrinsic_works`

Reply:
Added ticket in our internal issue tracker: https://linear.app/snowfork/issue/SNO-811